src/drag-and-drop/components/drop-zone-preview.scss

```diff
@@ -0,0 +1,12 @@
+@import "../../shared/styles/helpers";
+
+$border-width: 4px;
+.dropZonePreview {
+  border: $border-width dotted #144dec;
+  box-shadow: 0 0 12px rgba(0,0,0,0.25);
+  transform: translate(-$border-width, -$border-width);
```

Comment:
This is interesting.  What was this for?  Was this part of what was causing that problem with the border outline and the position?

Reply:
I guess @eireland based this on my older styles for regular item preview.
I think this transform rule was ensuring that preview with border matched the position of the actual item with border.
Otherwise, the element was shifted by these 4 pixels and there was a little image position jump on drag start.

Reply:
I took it out but can be put it back in if we want the jump.

Reply:
I would leave in whatever looks best.  I was just curious about the code.